stratego.build/src/main/java/mb/stratego/build/strincr/Frontend.java

```diff
@@ -705,9 +676,8 @@ private void stratFrontEnd(ExecContext execContext, String projectName, final Fi
                 internalStrats.add(strategyName);
             }
         }
-        final @Nullable File boilerplateFile =
-            resourceService.localPath(CommonPaths.strSepCompBoilerplateFile(location, projectName, moduleName));
-        assert boilerplateFile != null : "Bug in strSepCompBoilerplateFile or the arguments thereof: returned path is not a file";
+        // final ResourcePath boilerplateFile = CommonPaths.strSepCompBoilerplateFile(location, projectName, moduleName); // GK: boilerplateFile is not used
+        //assert boilerplateFile != null : "Bug in strSepCompBoilerplateFile or the arguments thereof: returned path is not a file"; // GK: strSepCompBoilerplateFile cannot return null
```

Comment:
Yeah, never noticed this was still here because the assert was still using the variable. This is from a time when the compiler still wrote intermediate results to file. You can remove this.